players/installing-mods.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Installing Mods
+description: A step by step guide on how to install mods for Fabric.
+---
+
+# Installing Mods
+
+This guide will walk you through installing mods for Fabric using the Minecraft Launcher.
+
+For third party launchers, you should consult their documentation.
+
+## 1. Download the Mod
+
+::: warning
+You should only download mods from sources you trust, for more information on finding mods, see the [Finding Mods](./finding-mods.md) guide.
+:::
+
+The majority of mods require [Fabric API](https://modrinth.com/mod/fabric-api) as well, so you should download that too.
+
+When downloading mods, ensure that:
+
+- They work on the version of Minecraft you want to play on.
+- They are for Fabric, not NeoForge, Rift, or any other mod loader.
```

Comment:
```suggestion
- They are for Fabric, not NeoForge, Spigot/Paper, or any other mod loader.
```

No version in which we support, had a working Rift release from what I can remember.